collections/_article/godot-xr-community-nov-2024.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Godot XR community - November 2024"
+excerpt: "Interesting projects from the Godot XR community."
+categories: ["news"]
+author: Bastiaan Olij
+image: /storage/blog/covers/november-2024-xr-community-update.webp
+image_caption_description: "tbd"
+date: 2024-11-01 12:00:00
+---
+
+Godot has a very active XR-focused community that mainly help each other out through the XR channel of the [official Godot Discord server](/community/).
+Today, we want to put a spotlight on a number of community projects.
+
+## Godot XR community Game Jams
+
+To showcase the engine capabilities and the ingenuity of the Godot XR community, game jams are now being organized twice a year.
+For participants, it is a fun way to get introduced to Godot's XR capabilities.
+
+### Godot XR Game Jam #1
+
+The [first game jam](https://itch.io/jam/godot-xr-game-jam) was held from December 1st until December 4th 2023.
+The theme of this game jam was Vapor.
+We saw both many new faces and long time community members join this first jam, with a total of 17 entries.
+
+The #1 spot went to [Cloud Shepherd](https://itch.io/jam/godot-xr-game-jam/rate/2402526) which won 3 out of 6 categories.
+This entry let you herd sheep on clouds in the sky. With a unique movement mechanic and using a fan to create
+a breeze with which to move the sheep around, this was a fascinating entry.
+
+The #2 spot went to [Super Steam Hands](https://itch.io/jam/godot-xr-game-jam/rate/2402404) which won 2 out of 6 categories.
+Don't let the thumbnail of this entry fool you. A great game where you can feel like Iron Man by using jets of steam to parkour around a level.
+
+The #3 spot went to [Pipe Stream](https://itch.io/jam/godot-xr-game-jam/rate/2400974) which won 1 out of 6 categories.
+A great little puzzle game where you need to chip away a path for vapour to reach a target. Very challenging!
+
+*Create a video for these entries, or each separate entry and embed here?*
+
+There were far more great entries so be sure to check out the jam page for more!
+
+### Godot XR Game Jam #2
+
+The [second game jam](https://itch.io/jam/godot-xr-game-jam-july-2024) was held from the 8th of July until the 15th of July 2024.
+We decided to give participants a full week to complete their game.
+The theme this jam was Relaxation. There were a total of 26 entries and this time we scored them on the average points in all categories.
+
+The #1 spot went to [Who's a good boy](https://itch.io/jam/godot-xr-game-jam-july-2024/rate/2821459) with 4.042 points.
+This cute little game has you play fetch with a dog in a park.
+This features some excellent interactions both from the player point of view, as well as the dog.
+
+The #2 spot went to [Constellation Coach VR](https://itch.io/jam/godot-xr-game-jam-july-2024/rate/2830514) with 3.974 points.
+This game has you find constellations in the night sky.
+By holding up a picture of a constellation, excellent use of haptic feedback allows you to find the relevant stars.
```

Comment:
Excellent haptic feedback lets you know when you've gotten close, and successfully aligning the constellation lights it up.